controller/server/controller.go

```diff
@@ -351,45 +348,30 @@ func (c *Controller) Run() {
 	}()
 }
 
```

Comment:
This is used in ziti-controller, assuming that's being updated

Reply:
It is. I have a PR sitting waiting for this to go through

Reply:
Merged now.